<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="zLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>90</y>
     <width>60</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans Narrow</family>
     <pointsize>60</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 60pt &quot;Liberation Sans Narrow&quot;;</string>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="xLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>60</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 60pt &quot;Liberation Sans Narrow&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="xSecondaryDro">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    border-style: solid;
    border-color: rgb(96, 96, 97);
    border-width: 1.5px;
    border-radius: 8px;
    color: white;
    background: rgb(86, 186, 187);
	font: 16pt &quot;Noto Sans Mono&quot;;
}</string>
   </property>
   <property name="text">
    <string>+0000.000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="zSecondaryDro">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    border-style: solid;
    border-color: rgb(96, 96, 97);
    border-width: 1.5px;
    border-radius: 8px;
    color: white;
    background: rgb(86, 186, 187);
	font: 16pt &quot;Noto Sans Mono&quot;;
}</string>
   </property>
   <property name="text">
    <string>+0000.000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="xPrimaryDro">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>305</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 40pt &quot;Noto Sans Mono&quot;;
    border-style: solid;
    border-color: rgb(10, 10, 10);
    border-width: 1.5px;
    border-radius: 8px;
    color: rgb(10, 10, 10);;
    background: rgb(217, 217, 217);
}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>+0000.000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="zPrimaryDro">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>305</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 40pt &quot;Noto Sans Mono&quot;;
    border-style: solid;
    border-color: rgb(10, 10, 10);
    border-width: 1.5px;
    border-radius: 8px;
    color: rgb(10, 10, 10);;
    background: rgb(217, 217, 217);
}</string>
   </property>
   <property name="text">
    <string>+0000.000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="zAbsRel">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>95</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    height: 40px;
    width: 40px;
    font: 14pt &quot;DejaVu Sans&quot;;
}</string>
   </property>
   <property name="text">
    <string>A/R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="xZero">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>15</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    height: 40px;
    width: 40px;
    font: 14pt &quot;DejaVu Sans&quot;;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="xAbsRel">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>15</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    height: 40px;
    width: 40px;
    font: 14pt &quot;DejaVu Sans&quot;;
}</string>
   </property>
   <property name="text">
    <string>A/R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zZero">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>95</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    height: 40px;
    width: 40px;
    font: 14pt &quot;DejaVu Sans&quot;;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="zUnit">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>115</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Liberation Mono&quot;;</string>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="xUnit">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>35</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Liberation Mono&quot;;</string>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="MDIButton" name="mdibutton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Set Datum</string>
   </property>
   <property name="rules" stdset="0">
    <string>[{&quot;name&quot;: &quot;enable set datum&quot;, &quot;property&quot;: &quot;Enable&quot;, &quot;expression&quot;: &quot;ch[0] != 0&quot;, &quot;channels&quot;: [{&quot;url&quot;: &quot;status:tool_in_spindle&quot;, &quot;trigger&quot;: true}]}]</string>
   </property>
   <property name="MDICommand" stdset="0">
    <string>G10 L20 P0 Z0</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MDIButton</class>
   <extends>QPushButton</extends>
   <header>qtpyvcp.widgets.button_widgets.mdi_button</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
